### PR TITLE
[v3] Trait Lifecycle Hooks

### PR DESCRIPTION
**Pull request by TWithers**
Trait hooks are `hookTraitName()` and not `traitNameHook()`

### CODE REVIEW

File: docs/lifecycle-hooks.md [hunk 1]
```diff
@@ -311,42 +311,42 @@ trait HasPostForm
...
     }
 
-    public function hasPostFormRendered()
+    public function renderedHasPostForm()
     {
         // ...
     }
 
-    public function hasPostFormDehydrated()
+    public function dehydratedHasPostForm()
```
Comment (PerryvanderMeer) [inline docs/lifecycle-hooks.md:349]: Should be `dehydrateHasPostForm`, as the hook method is `dehydrate()`.
See https://github.com/livewire/livewire/blob/ad33d14f9ca37a74f270b2495ec8f99aa3e29ba0/src/Features/SupportLifecycleHooks/SupportLifecycleHooks.php#L112C37-L112C37

Comment (PerryvanderMeer) [inline docs/lifecycle-hooks.md:349]: Should be `dehydrateHasPostForm`, as the hook method is `dehydrate()`.

See https://github.com/livewire/livewire/blob/main/src/Features/SupportLifecycleHooks/SupportLifecycleHooks.php#L112C37-L112C37

**Review by PerryvanderMeer — CHANGES_REQUESTED** [review]

File: docs/lifecycle-hooks.md (hunk 1 above)
Comment (joshhanley) [reply docs/lifecycle-hooks.md]: @PerryvanderMeer thanks, I have updated.

**Comment by joshhanley** [comment]
@TWithers thanks!

**Pull request merged**